api/src/db/networkRevenueProvider.ts

```diff
@@ -164,7 +179,7 @@ async function getDailyRevenue() {
     uakt: x.uakt,
     akt: uaktToAKT(x.uakt, 6),
     uusdc: x.uusdc,
-    usdc: udenomToDenom(x.uusdc),
+    usdc: udenomToDenom(x.uusdc, 6),
     usd: uaktToAKT(x.uakt, 6) * x.aktPrice + udenomToDenom(x.uusdc),
```

Comment:
`udenomToDenom(x.uusdc, 6)`?

Reply:
It's to match the precision of the akt amount